test/clearingHouse/ClearingHouse.badDebt.test.ts

```diff
@@ -94,25 +113,26 @@ describe("ClearingHouse badDebt", () => {
                 // bob short 2 ETH to reduce position
                 // exchanged notional: 1.655
                 // realized PnL: 1.655 - 2/7.866 * 800 = -201.7520684
-                // account value: 100 + 5.866 * 103.222 - 800 + 1.65 = -92.850 (bad debt)
+                // account value: 100 + 5.866 * 101.025123697070076996 - 800 + 1.65 = -105.736624393 (bad debt)
                 await expect(b2qExactInput(fixture, bob, "2", baseToken.address)).to.be.revertedWith("CH_NEMRM")
             })
 
             it("cannot reduce when not resulting bad debt but with not enough collateral", async () => {
                 // bob short 0.5 ETH to reduce position
                 // exchanged notional: 0.4139
                 // realized PnL: 0.4139 - 0.5/7.866 * 800 = -50.438
-                // account value: 100 + 7.366 * 103.222 - 800 + 0.4139 = 60.75 (no bad debt)
+                // account value: 100 + 7.366 * 101.025123697070076996 - 800 + 0.4139 = 44.5649611526 (no bad debt)
                 // free collateral: 100 - 50.438 - (800 - 0.4139 + 50.438) * 10% = -35.440
                 await expect(b2qExactInput(fixture, bob, "0.5", baseToken.address)).to.be.revertedWith("CH_NEFCI")
             })
 
             it("can reduce when not resulting bad debt and has enough collateral", async () => {
                 // bob short 0.1 ETH to reduce position
                 // exchanged notional: 0.083
-                // bob's realized PnL: 0.083 - 0.1/7.866 * 800 = -10.087
-                // bob's account value: 100 + 7.766 * 103.222 - 800 + 0.083 = 101.705 (no bad debt)
-                // bob's free collateral: 100 - 10.087 - (800 - 0.083 + 10.087) * 10% = 8.9126 > 0
+                // bob's realized PnL: 0.083 - 0.05/7.866 * 800 = -10.087
```

Comment:
Why is here `0.05/7.866` instead of the original `0.1/7.866`?
Isn't the param `0.1` in line 136?

Reply:
fixed in 558a8d2db7c402d809df779c2a463cd5def4c129